pkg/proc/proc.go

```diff
@@ -225,7 +226,15 @@ func Continue(dbp Process) error {
 				}
 				return conditionErrors(threads)
 			case g == nil || dbp.Common().fncallForG[g.ID] == nil:
-				// a hardcoded breakpoint somewhere else in the code (probably cgo)
+				// a hardcoded breakpoint somewhere else in the code (probably cgo), or manual stop in cgo
+				if !curthread.Arch().BreakInstrMovesPC() {
```

Comment:
Let's be consistent with how we access Arch as it's likely going to change in the future.

For now just store it as `arch := dbp.BinaryInfo().Arch` and use that instead of always calling `curthread.Arch()` or `dbp.BinInfo().Arch`.